src/docker-osx-dev

```diff
@@ -992,28 +992,32 @@ function instructions {
 #
 function load_paths_from_docker_compose {
   local readonly yaml_file_path="$1"
-  local in_volumes_block=false
   local paths=()
 
-  if [[ -f "$yaml_file_path" ]]; then
-    local line=""
-    while read line; do
-      if $in_volumes_block; then
-        if [[ "${line:0:2}" = "- " ]]; then
-          local readonly path=$(echo $line | sed -ne "s/- \([^:]*\):.*$/\1/p" | sed -e 's/^["'\'']//'  -e 's/["'\'']$//')
-          if [ ! -z "$path" ]; then
-            paths+=("$path")
-          fi
-        else
-          in_volumes_block=false
+  local yaml_file_param=""
+  if [ ! -z "${yaml_file_path}" ]; then
```

Comment:
I did not study the whole call chain so I expected the parameter to be empty when not using the -c argument. Is it?

I think it should as that will allow docker-compose to fall back to its defaults and environment variables.